cypress/integration/non-regression-tests-suite/mailinglist-signup-flow.spec.js

```diff
@@ -186,37 +187,11 @@ describe(`Feature - Mailing list sign up : Tests execution date and time : ${new
 		mailingListSignUp.getNextStep().click();
 		cy.degreeStage("Yes, I already have a degree");
 		mailingListSignUp.getNextStep().click();
-		cy.howCloseAreYou("I’m fairly sure and exploring my options");
-		mailingListSignUp.getNextStep().click();
-		mailingListSignUp
-			.getSubjectToTeach()
-			.select(this.mailingListTestData.whichSubjectdoYouWantToTeach);
-		mailingListSignUp.getNextStep().click();
-		mailingListSignUp.getPostcode().type(this.mailingListTestData.postCode);
-		mailingListSignUp.getNextStep().click();
-		cy.acceptPrivacyPolicy();
-		mailingListSignUp.getCompleteSignUpButton().click();
-		mailingListSignUp.getContent().should("have.text", `You've signed up`);
-		cy.contains("Speak to us")
-			.next()
-			.children()
-			.as("speakToUsSection")
-			.should("exist")
-			.invoke("attr", "href")
-			.then(function (targetLink) {
-				expect(targetLink).to.equal("tel:08003892501");
-			});
-		cy.get("@speakToUsSection").then(function (phoneNumber) {
-			cy.log();
-			expect(phoneNumber.text()).to.equal("0800 389 2501");
-		});
-		cy.contains("a", "feedback on this website").invoke("removeAttr", "target").click();
-		cy.get(".freebirdFormviewerViewHeaderTitle")
-			.should("exist")
-			.should("have.text", "Get into Teaching: Feedback Survey");
+		cy.mailingListSignUpBar().click();
+		cy.location("pathname").should("equal", Navlinks.mailingListSignup);
 	});
 
-	it('It hides the feedback bar if user clicks on "Hide" link', function () {
+	it('It hides the mailing list bar if user clicks on "Not now" link', function () {
 		let rnum = Math.floor(Math.random() * 1000000000 + 1);
```

Comment:
Ditto for this one - I think the test would be better as something like:

- Visit 3 pages
- Banner appears
- Click sign up takes you to ML sign up

- Navigate 3 pages
- Banner appears
- Click dismiss, banner disappears
- Navigate another page, banner stays gone